packages/generators/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@webpack-cli/generators",
-  "version": "3.0.7",
+  "version": "3.0.8",
```

Comment:
This should be updated only while releasing the package.

Reply:
Remove this